### ISSUE TITLE
Untranslated plural strings, (and formatted strings?) cause errors

### ISSUE THREAD

**Issue by dreinn** [nvaccess/nvda#20351]
### Brief summary

With NVDA 2026.2beta2 and beta3, pressing nvda-shift-b produces no results, and an error emerges in the logs

### Steps to reproduce

with nvda 2026.2beta3, press nvda-shift-b

### Actual behavior

no feedback, only error

[batteryError.txt](https://github.com/user-attachments/files/29011038/batteryError.txt)

### Expected behavior

NVDA should announce battery status

### NVDA logs, crash dumps and other attachments

the log is found in the uppermost fields as an attachment

### NVDA type

installed copy

### NVDA version

2026.2beta3

### Windows version

windows 11,  10.0.26200.8655] 

### Does the issue still occur after restarting your computer?

I have not restarted my computer

### If NVDA add-ons are disabled, is your problem still occurring?

I have not restarted NVDA with add-ons disabled

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have not run the System Accessibility Repair Tool

**Comment by CyrilleB79** [comment]
Hi @dreinn. I imagine you use NVDA in Italian.

The error is caused by a missing plural form in the translation file:
```
#. Translators: This is presented to inform the user of a progress bar percentage.
#: NVDAObjects\behaviors.py:83 winAPI\_powerTracking.py:217
#, python-format
msgid "%d percent"
msgid_plural "%d percent"
msgstr[0] "%d percento"
msgstr[1] ""
```

You have to fix this in the Italian .po file.
Though, I'd recommend to keep this issue open in case NV Access wants to have translations more robust. It's not desirable to have an error when are missing: NVDA should fallback to English instead. If not possible, such translation files should be rejected with error when submitted.

**Comment by dreinn** [comment]
Oh my God... Thank you very much!
I fixed it just now and I have uploaded the new translation.
Yes,  I will keep the issue opened, because NVDA should'nt give errors in this situation. At least, it should provide the information in English, it would also be easier to diagnose.

**Comment by makhlwf** [comment]
> The detail that stands out is that this affects NVDA+Shift+B in both 2026.2beta2 and beta3: instead of announcing battery status, NVDA only logs an error. `source/winAPI/_powerTracking.py` is the part I would inspect first, especially the plural/formatted battery status messages and how untranslated strings are formatted. For a PR, I would start by tightening that formatting path and adding coverage so untranslated plural strings fall back cleanly.

The issue has been fixed so you don't need to edit the specified file unless you would make a solution that will affect other files since this issue could happen anywhere that plural forms are used and not only in the battery related code for announcements
